WooCommerce/Classes/View Modifiers/View+AddingDividers.swift

```diff
@@ -0,0 +1,20 @@
+import SwiftUI
+
+/// Custom view modifer for adding dividers on top and bottom of a view.
+struct TopAndBottomDividers: ViewModifier {
```

Comment:
This would be a time saver! I wonder if we could add a parameter later to ignore safe areas 🤔